empress/support_files/js/colorer.js

```diff
@@ -175,15 +175,21 @@ define(["chroma", "underscore", "util"], function (chroma, _, util) {
             );
         }
 
+        var interpolator;
+        if (this.color === Colorer.__PR_OR_GR) {
+            interpolator = chroma.scale(Colorer.__prOrGr).domain([0, 1]);
+        } else {
+            interpolator = chroma.scale(this.color).domain([0, 1]);
+        }
+
         if (this.sortedUniqueValues.length === 1) {
             // If there's only 1 unique value, set its color as the first in
```

Comment:
```suggestion
            interpolator = interpolator.domain([0, 1]);
            // If there's only 1 unique value, set its color as the first in
```

Technically I don't even think we need to add this -- the default domain of a scale in chroma is [0, 1] (see [here](https://gka.github.io/chroma.js/#chroma-scale)). However, I do like explicitly setting it as is currently done in this PR, both in case chroma'd change this in the future and to make the code more understandable.

Reply:
@fedarko thanks for reviewing this! I think I have addressed all of your comments.